app/src/pages/movies/movie/components/EditMovieForm.jsx

```diff
@@ -27,27 +28,44 @@ const ButtonContainer = styled.div`
 `
 
 function EditMovieForm(props) {  
-    const [genres, setGenres] = useState([]);
-    
-    useEffect(() => {
-        Promise
-            .resolve(genresFromJson)
-            .then(genres => setGenres(genres))
-    }, [])
+    const [title, setTitle] = useState(props.movie.title);
+    const [release_date, setReleaseDate] = useState(props.movie.release_date);
+    const [poster_path, setUrl] = useState(props.movie.poster_path);
+    const [genres, setGenres] = useState(props.movie.genres.join(','));
+    const [overview, setOverview] = useState(props.movie.overview);
+    const [runtime, setRuntime] = useState(props.movie.runtime);
+
+    function confirmUpdate(event){
+        event.preventDefault();
+        var movie = {

```

Comment:
updates can be in the action

Reply:
Will be done in the next [HW#7](https://github.com/aldaniluk/react-mentoring/pull/7)